generic-template/Cargo.toml

```diff
@@ -25,6 +25,9 @@ serde = { version = "1.0.197", default-features = false }
 serde_json = "1.0.121"
 smallvec = "1.11.0"
 
+#polkadot-runtime-wrappers = { git = "https://github.com/OpenZeppelin/polkadot-runtime-wrappers.git", branch = "amar-oz-system-wrapper-poc", default-features = false }
+polkadot-runtime-wrappers = { path = "../../polkadot-runtime-wrappers", default-features = false }
```

Comment:
I know this probably got pushed accidentally, but I'm just pointing it so we don't forget